.npmignore

```diff
@@ -1,4 +1,8 @@
-**/*
+# Ignore all files
+*
+
+# Add exceptions
 !js/
+!js/**/*
 !CONTRIBUTING.MD
```

Comment:
Not sure if we _really_ need `js/test/**` included in the published artifact, but it only adds ~50kb so I didn't bother re-ignoring it.